common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -183,7 +183,7 @@ trait PerformanceSwitches {
     "If switched on, explicitly request lazy loading of images on supporting browsers.",
     owners = Seq(Owner.withGithub("nicl")),
     safeState = On,
-    sellByDate = LocalDate.of(2029, 9, 3),
+    sellByDate = never,
```

Comment:
Introduced in https://github.com/guardian/frontend/pull/21584/files#diff-d05c492cbdd04e6bd2f8c1323c2e775bf74ce0b8675fd9dbb39ba9d8847f373eR196, it does not seem like this was supposed to ever expire.